backend/src/modules/Boards/controller/Boards.controller.spec.js

```diff
@@ -0,0 +1,31 @@
+import { agent } from 'supertest';
+import { testApi } from '../../../testUtils/testApi.js';
+import { BoardsController } from './BoardsController.js';
+
+describe('BoardsController |', () => {
+  const boardBody = {
+    boardName: 'Wroclaw',
+    coords: { latitude: 51.88569995139321, longitude: 17.02390643626451 },
+    accessType: 'public',
+    adminId: '507f191e810c19729de860ea',
+  };
+
+  test('POST /rest-api/boards | when return added body', async () => {
+    // GIVEN
+    const testService = {
+      addBoard: async (boardBody) => boardBody,
+    };
+    const boardsController = new BoardsController(testService);
+    const app = testApi('/rest-api', boardsController.router);
+
+    // WHEN
+    const { body, status } = await agent(app).post('/rest-api/boards').send(boardBody);
+
+    // THEN
+    expect(status).toEqual(200);
+    expect(body.returnedData.boardName).toEqual(boardBody.boardName);
+    expect(body.returnedData.coords).toEqual(boardBody.coords);
+    expect(body.returnedData.accessType).toEqual(boardBody.accessType);
+    expect(body.returnedData.adminId).toEqual(boardBody.adminId);
+  });
+});
```

Comment:
Przyda się jeszcze test na Fail.
Bardzo ładnie zrobił to @PSwiecicki  - zerknij na jego PR :)

https://github.com/PiotrRynio/CodersCamp2021.Project-2.FullStack.React-Node/pull/153